db/migrate/20151106172531_remove_uniqueness_display_name_index_users.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveUniquenessDisplayNameIndexUsers < ActiveRecord::Migration
+  def change
+    remove_index :users, column: :display_name, unique: true
```

Comment:
Can you think of another case where we'd need this? It'd be nice not to have to fix any conflicts if we remove this validation.